### ISSUE TITLE
Tables created without primary key

### ISSUE THREAD

**Issue by jakudlaty** [JetBrains/Exposed#695]
I have tried to create two simple tables with reference, and i have found a bug:

When i try to execute code: 
```
object Products : Table() {
    val id = integer("id").autoIncrement().primaryKey()
}

object ProductDescriptions : Table() {
    val id = integer("id").autoIncrement().primaryKey()
    val productId = reference("product_id", Products.id)
}

transaction {
   SchemaUtils.create(Products)
   SchemaUtils.create(ProductDescriptions)
}
```
everithing works perfectly, created DDL i correct:

```
CREATE TABLE IF NOT EXISTS products (id SERIAL PRIMARY KEY)
CREATE TABLE IF NOT EXISTS productdescriptions (id SERIAL PRIMARY KEY, product_id INT NOT NULL, CONSTRAINT fk_productdescriptions_product_id_id FOREIGN KEY (product_id) REFERENCES products(id) ON DELETE RESTRICT ON UPDATE RESTRICT)
```
But when i switch order of invocation of ```autoIncrement()``` and ```primaryKey()``` exposed is generating wrong DDL (without ```PRIMARY KEY``` keywords)

For example:
```
object Products : Table() {
    val id = integer("id").primaryKey().autoIncrement()
}

object ProductDescriptions : Table() {
    val id = integer("id").primaryKey().autoIncrement()
    val productId = reference("product_id", Products.id)
}
```
 will generate the following SQL:
```
CREATE TABLE IF NOT EXISTS products (id SERIAL NOT NULL)
CREATE TABLE IF NOT EXISTS productdescriptions (id SERIAL NOT NULL, product_id INT NOT NULL, CONSTRAINT fk_productdescriptions_product_id_id FOREIGN KEY (product_id) REFERENCES products(id) ON DELETE RESTRICT ON UPDATE RESTRICT)
```
Whitch leads to the following error on postgresql database:
```
ERROR: there is no unique constraint matching given keys for referenced table "products"
```

Exposed version affected: ```'org.jetbrains.exposed:exposed:0.17.7'```

**Comment by Tapac** [comment]
@jakudlaty , could you please specify a database on which this bug happens. Also, could you please check is it reproducible on 0.18.1?

**Comment by Tapac** [comment]
Seems it was already fixed in dfe6db7c commit in master and wasn't cherry-picked to 0.17.x branch.

**Comment by jakudlaty** [comment]
I have reproduced it successfully on postgresql 11, postgresql 12.1 on exposed 0.17.7
Everyting works great on 0.18.1
